waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -6,18 +6,18 @@ where
 import Control.Monad.Except (throwError)
```

Comment:
I introduced a lot of changes to this file. The best way to review it is probably to avoid looking at the diff altogether.